cluster/gce/util.sh

```diff
@@ -595,6 +595,17 @@ function create-node-template() {
       done
     fi
   fi
+  #add in optional ones
+  if [[ ! -z ${NODE_LOCAL_SSDS_EXT-""} ]]; then
+    IFS=";" read -r -a ssdgroups <<< ${NODE_LOCAL_SSDS_EXT}
+    for ssdgroup in "${ssdgroups[@]}"
+    do
+      IFS="," read -r -a ssdopts <<< ${ssdgroup}
+      for i in $(seq ${ssdopts[0]}); do
```

Comment:
What happens if the user misformats these variables? (like maybe only includes 1 out of the 3 values)

Reply:
it just kinda "goes with it" and tries its best, an invalid array access here will basically just return the empty string. Should I add some validation?

Reply:
I think some validation would be good, so that we don't accidentally create more local SSDs than the user intended.  Actually, there was a bug a few months ago where "seq 0" returns "1 0" on mac, so it actually created 2 local SSDs instead of 0. We should make sure we don't hit the same issue here.

I guess if the local ssd interface is malformed, then the instance creation should fail with invalid argument.